packages/react-text/src/components/Text/Text.test.tsx

```diff
@@ -6,4 +10,160 @@ describe('Text', () => {
+    `);
+  });
+
+  it('applies the underline style', () => {
+    const { getByText } = render(<Text underline>Test</Text>);
+
+    const textElement = getByText('Test');
+    expect(textElement).toHaveStyle(`
+      text-decoration: underline;
+    `);
+  });
+
+  it('applies the strikethrough style', () => {
+    const { getByText } = render(<Text strikethrough>Test</Text>);
+
+    const textElement = getByText('Test');
+    expect(textElement).toHaveStyle(`
+      text-decoration: line-through;
+    `);
+  });
+
+  it('applies both strikethrough and underline styles', () => {
+    const { getByText } = render(
+      <Text strikethrough underline>
+        Test
+      </Text>,
+    );
+
+    const textElement = getByText('Test');
+    expect(textElement).toHaveStyle(`
+      text-decoration: line-through underline;
+    `);
+  });
+
+  it.each([
+    [100, 'base', '100'],
+    [200, 'base', '200'],
+    [300, 'base', '300'],
+    [400, 'base', '400'],
+    [500, 'base', '500'],
+    [600, 'base', '600'],
+    [700, 'hero', '700'],
+    [800, 'hero', '800'],
+    [900, 'hero', '900'],
+    [1000, 'hero', '1000'],
+  ])('applies the %s token sizing styles', (sizeToken: any, expectedPrefix, expectedValue) => {
+    const { getByText } = render(<Text size={sizeToken}>Test</Text>);
+
+    const textElement = getByText('Test');
+    expect(textElement).toHaveStyle(`
+      font-size: var(--global-type-fontSizes-${expectedPrefix}-${expectedValue});
+      line-height: var(--global-type-lineHeights-${expectedPrefix}-${expectedValue});
+    `);
+  });
+
+  it.each([
+    ['base', 'base'],
+    ['monospace', 'monospace'],
+    ['numeric', 'numeric'],
+  ])('applies %s font', (input: any, expectedValue) => {
+    const { getByText } = render(<Text font={input}>Test</Text>);
+
+    const textElement = getByText('Test');
+    expect(textElement).toHaveStyle(`
+      font-family: var(--global-type-fontFamilies-${expectedValue});
+    `);
+  });
+
+  it.each([
+    ['regular', 'regular'],
+    ['medium', 'medium'],
+    ['semibold', 'semibold'],
+  ])('applies %s weight', (input: any, expectedValue) => {
+    const { getByText } = render(<Text weight={input}>Test</Text>);
+
+    const textElement = getByText('Test');
+    expect(textElement).toHaveStyle(`
+      font-weight: var(--global-type-fontWeights-${expectedValue});
+    `);
+  });
+
+  it.each([
+    ['start', 'start'],
+    ['center', 'center'],
+    ['end', 'end'],
+    ['justify', 'justify'],
+  ])('applies a %s alignment', (input: any, expectedValue) => {
+    const { getByText } = render(<Text align={input}>Test</Text>);
+
+    const textElement = getByText('Test');
+    expect(textElement).toHaveStyle(`
+      text-align: ${expectedValue};
+    `);
+  });
+
+  it('allows overriding the component being rendered', () => {
+    const { getByText } = render(<Text as="p">Test</Text>);
+
+    const textElement = getByText('Test');
+    expect(textElement.nodeName).toBe('P');
+  });
```

Comment:
Are you testing something specific there? As this is covered by our conformance tests:

https://github.com/microsoft/fluentui/blob/8de04ed56820cfdb7c13d9c108b0eae596cd3215/packages/react-conformance/src/defaultTests.tsx#L433-L461